### DISCUSSION TITLE
GitHub student devloper pack

### DISCUSSION THREAD

**Discussion by Aurosish07** [community/community#46913] [GitHub Education]
I have requested in GitHub student devloper pack , i am in BSC 1ST YE (INFORMATION TECHNOLOGY) it's been 6hrs still my application is pending should I wait more or reapply

**Comment by jge162** [comment]
Application takes 24/48 hours. Mine personally took 36 hours. So 6 hours is not enough time.

**Reply by jge162** [reply]
They will email you once you’re approved so no need to worry.

**Reply by Aurosish07** [reply]
Ohh! Thank you i didn't knew actually how much time it took.

**Reply by jge162** [reply]
no worries, once you get acces enjoy. My favorite perk is copilot...

**Comment by Aurosish07** [comment]
Yaa , i got my student Devloper pack now . Omg thank you github

**Reply by jge162** [reply]
Sweet. Enjoy!!

**Reply by samus-aran** [reply]
Congratulations! 🎉 
@Aurosish07 What do you first plan to do with the benefits? Any particular offer you were drawn to?

**Reply by Aurosish07** [reply]
Ya , i will use the domain in my website https://aurosishnet.000webhostapp.com/

**Reply by samus-aran** [reply]
Awesome! That's a great start 👏 
Keep us informed of any other benefits you enjoy and show us how you get on throughout your education! 💻
